tcping.go

```diff
@@ -138,26 +190,39 @@ func processUserInput(tcpStats *stats) {
 	args := flag.Args()
 	nFlag := flag.NFlag()
 
+	// we need to set printers first, because they're used for
+	// errors reporting and other output.
+	if *outputJson {
+		tcpStats.printer = newJsonPrinter(*prettyJson)
+	} else {
+		tcpStats.printer = &planePrinter{}
+	}
+
+	if *prettyJson && !*outputJson {
+		tcpStats.printer.printError("--pretty has no effect without the -j flag.")
```

Comment:
This line errors out in case of running the program like:

```bash
go run . example.com 443 --pretty
```

Reply:
Yes, because it was [requested](https://github.com/pouriyajamshidi/tcping/pull/74#discussion_r1190151088) to be like that

Reply:
I mean the program crashes 😄 It does not show the `usage()`

Reply:
ah hold on! some caching issues on my test pod... it is working fine.